### PR TITLE
Add temporary repartitions until bugfix

### PR DESCRIPTION
**Pull request by klaricch**

### CODE REVIEW

File: gnomad_constraint/pipeline/constraint_pipeline.py [hunk 1]
```diff
@@ -383,9 +383,13 @@ def main(args):
             # Build plateau and coverage models for autosomes/pseudoautosomal regions,
             # chromosome X, and chromosome Y
             for r in regions:
+                # TODO: Remove repartition once partition_hint bugs are resolved
```
Comment (jkgoodrich) [inline gnomad_constraint/pipeline/constraint_pipeline.py:386]:
```suggestion
                # TODO: Remove repartition once partition_hint bugs are resolved.
```

File: gnomad_constraint/pipeline/constraint_pipeline.py [hunk 2]
```diff
@@ -405,6 +409,12 @@ def main(args):
             res = resources.apply_models
             res.check_resource_existence()
 
+            # TODO: Remove repartition once partition write bugs are resolved
```
Comment (jkgoodrich) [inline gnomad_constraint/pipeline/constraint_pipeline.py:412]:
```suggestion
            # TODO: Remove repartition once partition write bugs are resolved.
```

File: gnomad_constraint/utils/constraint.py [hunk 3]
```diff
@@ -272,6 +272,9 @@ def create_observed_and_possible_ht(
         count_downsamplings=pops,
         use_table_group_by=True,
     )
+
+    # TODO: Remove repartition once partition_hint bugs are resolved
```
Comment (jkgoodrich) [inline gnomad_constraint/utils/constraint.py:276]:
```suggestion
    # TODO: Remove repartition once partition_hint bugs are resolved.
```

File: gnomad_constraint/utils/constraint.py [hunk 4]
```diff
@@ -467,6 +470,9 @@ def apply_models(
         .aggregate(**agg_expr)
     )
 
+    # TODO: Remove repartition once partition_hint bugs are resolved
```
Comment (jkgoodrich) [inline gnomad_constraint/utils/constraint.py:473]:
```suggestion
    # TODO: Remove repartition once partition_hint bugs are resolved.
```

File: gnomad_constraint/utils/constraint.py [hunk 5]
```diff
@@ -467,6 +470,9 @@ def apply_models(
...
+    ht = ht.repartition(expected_variant_partition_hint)
```
Comment (jkgoodrich) [inline gnomad_constraint/utils/constraint.py:474]: Just want to confirm that the ht doesn't need a repartition to `obs_pos_count_partition_hint` after the call to `create_observed_and_possible_ht`

**Review by jkgoodrich — CHANGES_REQUESTED** [review]
Mostly just add periods to end of comment

File: gnomad_constraint/utils/constraint.py (hunk 5 above)
Comment (klaricch) [reply gnomad_constraint/utils/constraint.py:474]: i think it will run fine with these edits but can make more changes if it fails

**Review by jkgoodrich — APPROVED** [review]
LGTM!

**Pull request merged**